libs/python/helperCommandExecution.py

```diff
@@ -9,8 +9,8 @@
 log = logging.getLogger(__name__)
 
 
-def runShellCommand(btpUsecase, command, format, info):
-    return runShellCommandFlex(btpUsecase, command, format, info, True, False)
+def runShellCommand(btpUsecase, command, format, info, exitIfError: bool = True):
```

Comment:
Basically `runShellCommand` and `runShellCommandFlex` are the same now except for one more boolean parameter that is usually `FALSE`.
Wouldn't it make more sense to introduce a new function `runShellCommandFlexandGetJsonresult` that delegates to `runShellCommandFlex` with the same function signature instead of "overloading" the `runShellCommand` function with optional parameters?

Reply:
Absolutely makes sense. I've implemented your proposal and added as well some additional log information to better explain what the code is doing around the app subscriptions and the different names (appName and commercialAppName).
Will run a "slim" integration test and ping you once it's done.